src/main/resources/templates/db_scripts/alterHridSequences.sql

```diff
@@ -0,0 +1,10 @@
+-- Increase the initial size of the HRID sequences
+
+ALTER SEQUENCE IF EXISTS ${myuniversity}_${mymodule}.hrid_instances_seq
+  AS BIGINT MAXVALUE 99999999999;
+
+ALTER SEQUENCE IF EXISTS ${myuniversity}_${mymodule}.hrid_holdings_seq
+  AS BIGINT MAXVALUE 99999999999;
```

Comment:
Since we are updating the sequences here, does it make sense to do [MODINVSTOR-406](https://issues.folio.org/browse/MODINVSTOR-406) as well?

Reply:
Make sense to me. 
>  We should determine if the sequences should be owned by this table for ease of clean up and either add the owned by to the holdings/items sequence creation or remove it from the instances sequence creation.

Do you have any concerns that it is not needed here?

Reply:
I am not sure what is the best practice for sequences. I assume that binding them to the HRID settings table is the right thing to do, but I am not a DBA, so I could be wrong. The previous HRID sequences were bound to the actual jsonb column of the inventory type they generated HRIDs for, since there were no HRID settings. I suspect we want to add the same owner as the instances sequence to the holdings and items sequences. I can't imagine we'd want to drop the owner from the instances sequence. But that is just my opinion. 😄

Reply:
Ok, I've add owner for the sequences.